packages/contracts-bedrock/scripts/SemverLock.s.sol

```diff
@@ -27,8 +27,28 @@ contract SemverLock is Script {
             commands[1] = _files[i];
             string memory fileContents = string(vm.ffi(commands));
 
+            // Grab the contract name
+            commands = new string[](3);
+            commands[0] = "bash";
+            commands[1] = "-c";
+            commands[2] = string.concat(
+                "echo \"",
+                _files[i],
+                "\"| sed -E \'s|src/.*/(.+)\\.sol|\\1|\'"
+            );
+            string memory contractName = string(vm.ffi(commands));
+
+            // Parse the artifact to get the contract's initcode hash.
+            bytes memory initCode = vm.getCode(string.concat(
+                "forge-artifacts/",
```

Comment:
This could be portable if we call out to forge config --json and pull the out field using jq

Reply:
Oo good idea. Added 🖖